src/types.ts

```diff
@@ -284,3 +284,22 @@ export interface Hook<T extends FlagValue = FlagValue> {
    */
   finally?(hookContext: Readonly<HookContext<T>>, hookHints?: HookHints): Promise<void> | void;
 }
+
+/**
+ * Transaction context is a mechanism for adding transaction specific context that
+ * is merged with evaluation context prior to flag evaluation. Examples of potential
+ * transaction specific context include: a user id, user agent, or IP address.
+ */
+export type TransactionContext = EvaluationContext;
```

Comment:
@toddbaert what do you think about this? We could use evaluation context but having a transaction concept type seemed nice.